helm-chart/renku-notebooks/values.yaml

```diff
@@ -48,27 +48,33 @@ debug: true
 ## Note that requests are also limits, i.e. a user's jupyter kernel
 ## that exceeds the requested memory limit will be terminated/restarted.
 serverOptions:
+  defaultUrl:
+    displayName: Default Environment
+    type: enum
+    default: /lab
+    ## add /rstudio to the list for R images
+    options: [/lab]
   cpu_request:
-    displayName: CPU request
+    displayName: Number of CPUs
     type: enum
     default: 0.5
     options: [0.5, 1.0]
   mem_request:
-    displayName: Memory request
+    displayName: Amount of Memory
     type: enum
     default: 1G
     options: [1G, 2G]
+  # Int values are also supported
+  # gpu_request:
+  #   displayName: Number of GPUs
+  #   type: int
+  #   default: 0
+  #   range: [0, 0]
   gpu_request:
```

Comment:
We could comment this out (and remove the already commented `gpu_request` above) and leave the setting in the dev python script, otherwise there is no clean way for the admin to remove the option completely. It may be desirable in environments where no GPUs are available at all, so the user can't even ask the admin to provide some.

Reply:
I think this should be here even if the user cannot make a choice because we need to inform the user about what resources they will get when they start an environment.

Reply:
After discussing with Rok, we have decided to leave in the option, even if it is 0.